ddtrace/internal/remoteconfig/_connectors.py

```diff
@@ -0,0 +1,101 @@
+from ctypes import c_char
+import json
+import os
+import sys
+from typing import Any
+from typing import Dict
+from typing import Mapping
+from uuid import UUID
+
+from ddtrace.internal.compat import PY2
+from ddtrace.internal.compat import to_unicode
+from ddtrace.internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+# Size of the shared variable. It's calculated based on Remote Config Payloads. At 2023-04-26 we measure on stagging
+# RC payloads and the max size of a multiprocess.array was 139.002 (sys.getsizeof(data.value)) and
+# max len 138.969 (len(data.value))
+SHARED_MEMORY_SIZE = 603432
```

Comment:
It would be great if we could have someone from @DataDog/remote-config to review this bit. In DI we would handle payloads up to 1 MB (and I'm checking with the team whether this is still the case). I don't remember if the RFC dictates any sort of hard payload size limit, but if it does, it's probably larger than this.

Reply:
I validated this size in this test: https://github.com/DataDog/system-tests/pull/1143 :+1: